metadata.yaml

```diff
@@ -12,7 +12,7 @@ resources:
   oci-image:
     type: oci-image
     description: Backing OCI image
-    upstream-source: docker.io/charmed/base-mlflow
+    upstream-source: docker.io/charmedkubeflow/mlflow:latest
```

Comment:
Should we use a tag here? Maybe we can keep a 1:1 relation with upstream in terms of image tags, in the end this image corresponds to a certain mlflow version, no?

Reply:
for now we have just latest as we need to adjust the image's CI .... we cant use upstream tag as we may want to have multiple versions of upstream tag in case we will fix vulnerabilities. There is a task in jira for that for now we have just latest tag.